store/src/test/java/com/nytimes/android/external/store/KeyParserTest.java

```diff
@@ -49,8 +49,11 @@ public void testStoreWithKeyParserFuncNoPersister() throws Exception {
         testObservable.assertNoErrors()
                 .assertValues(NETWORK + KEY)
                 .awaitTerminalEvent();
-        // TODO assertTrue(testObservable.isDisposed()); ??
 
+        // This check doesn't work in RxJava 2.0.7
```

Comment:
Its not necessary as `awaitTerminalEvent` ensures that the observer is in terminal state. And it's covered by RxJavas tests. So the comment can be removed completely.